### DISCUSSION TITLE
单测规范讨论

### DISCUSSION THREAD

**Discussion by fanhousanbu** [go-atomci/atomci#93] [General]
1. 是否要约束单测框架
`自带testing` `GoConvey` `testify`

2. 是否要增加徽章用于展示单测覆盖率

3. go先天对单测作了不少约定，因此项目中是否还要额外约束？

**Comment by sampsonye** [comment]
1. 看起来目前go自带的单测框架应该满足我们的需求了吧

2. 建议在`github actions`中增加单测流水线，并展示单测覆盖率

3. 额外的约束我们可以拉出来专门讨论吧，目前未想到有什么需要补充的

**Reply by colynn** [reply]
建议统一使用自带的单测module `testing`，　只是对于mock数据块　应该需要引入相应的工具吗，比如 https://github.com/golang/mock
